src/blueapi/service/interface.py

```diff
@@ -48,13 +49,17 @@ def worker() -> TaskWorker:
 
 
 @lru_cache
-def messaging_template() -> MessagingTemplate | None:
+def messaging_template() -> MessagingTemplate:
```

Comment:
```suggestion
def messaging_template() -> MessagingTemplate | None:
```

This still returns None if stomp_config is None?